lib/lte_link_control/lte_lc_helpers.c

```diff
@@ -1543,3 +1594,29 @@ int parse_periodic_search_pattern(const char *const pattern_str,
 
 	return 0;
 }
+
+int lte_lc_rai_set(void)
+{
+	int err;
+
+	if (IS_ENABLED(CONFIG_LTE_RAI_REQ)) {
+		LOG_DBG("Enabling RAI with notifications");
+	} else {
+		LOG_DBG("Disabling RAI");
+	}
+
+	err = nrf_modem_at_printf("AT%%RAI=%d", IS_ENABLED(CONFIG_LTE_RAI_REQ) ? 2 : 0);
+	if (err) {
+		if (IS_ENABLED(CONFIG_LTE_RAI_REQ)) {
+			LOG_DBG("Failed to enable RAI with notifications so trying without them");
+			/* If AT%RAI=2 failed, modem might not support it so using older API */
+			err = nrf_modem_at_printf("AT%%RAI=1");
+		}
+		if (err) {
+			LOG_ERR("Failed to configure RAI, err %d", err);
```

Comment:
```suggestion
			LOG_ERR("Failed to enable RAI, err %d", err);
```

Reply:
We do get here also when disabling is failing.